systolic/CgraMemRightAndBottomRTL.py

```diff
@@ -124,11 +124,21 @@ def construct(s, DataType, PredicateType, CtrlType, width, height,
         s.tile[i].to_mem_wdata.rdy //= 0
 
   # Line trace
-  def line_trace(s):
+  def line_trace( s ):
     # str = "||".join([ x.element.line_trace() for x in s.tile ])
     # str += " :: [" + s.data_mem.line_trace() + "]"
     res = "||\n".join([(("[tile" + str(i) + "]: ") + x.line_trace() + x.ctrl_mem.line_trace())
                        for (i,x) in enumerate(s.tile)])
     res += "\n :: SouthMem [" + s.data_mem_south.line_trace() + "]    \n"
     res += "\n :: EastMem [" + s.data_mem_east.line_trace() + "]    \n"
     return res
+
+
+  def verbose_trace( s, verbosity = 1 ):

```

Comment:
Remove the unnecessary space, (keep the one before and after `=`).

Reply:
done